.github/workflows/codeql.yml

```diff
@@ -0,0 +1,72 @@
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  schedule:
+    # Daily at 1:00.
+    - cron: '0 1 * * *'
```

Comment:
This is nightly, but the description says "only on sunday". Should this be '0 1 * * 0' ?

Reply:
We changed it to nightly after some discussion. It was initially weekly.